L1Trigger/L1TCaloLayer1/python/simCaloStage2Layer1Digis_cfi.py

```diff
@@ -18,5 +18,5 @@
     unpackEcalMask = cms.bool(False),
     unpackHcalMask = cms.bool(False),
     # See UCTLayer1.hh for firmware version
-    firmwareVersion = cms.int32(1),
+    firmwareVersion = cms.int32(3),
```

Comment:
Yes and no, the change will not break the ability to re-emulate using old data, but the bit-level agreement will be broken with old (bugged firmware) data.
So far, there has in general been no requirement that L1T emulation can reproduce historic data at bit-level automatically.  (the latest firmware and latest emulator should agree of course)